client/dashboard/domains/dataviews/actions.tsx

```diff
@@ -201,7 +211,19 @@ export const useActions = ( { user, site }: { user: User; site?: Site } ) => {
 				label: __( 'Change site address' ),
 				supportsBulk: false,
 				callback: () => {},
```

Comment:
Question: is this required? In the sense that can we just not pass callback if it's a noop.

Reply:
Yeah I noticed this in [an earlier PR](https://github.com/Automattic/wp-calypso/pull/104540#:~:text=The%20types%20say%20that%20closeModal%20can%20be%20undefined.%20In%20practice%20it%20never%20is%2C%20but%20those%20are%20the%20types). I believe this is just an incorrect type in the upstream package, because when you inspect the dataviews code you can see that it won't crash if a callback isn't supplied.

I see there's a `noop` defined at the top of this file. We could use it for all of these callbacks to make it explicit that, yes, we do know that we're doing nothing here.